ls1intum/Artemis
# `General`: Add missing details to course detail view in course management

### Checklist
#### General



#### Client


### Motivation and Context
There have been some details missing in the course detail page. This PR adds this missing information.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 course


1. Log in to Artemis
2. Navigate to Course Detail View
3. Verify that all necessary details of the course are displayed

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests


### Screenshots
![image](https://github.com/ls1intum/Artemis/assets/78978542/ecf5d83d-e83d-43c5-9b36-a5dff88fb858)
